src/main/java/com/splunk/kafka/connect/SplunkSinkTask.java

```diff
@@ -401,7 +401,7 @@ private Event createHecEventFrom(final SinkRecord record) {
                 event.setTied(record);
                 event.addFields(connectorConfig.enrichments);
             } catch(Exception e) {
-                log.trace("event does not follow correct HEC pre-formatted format: {}", record.value().toString());
+                log.trace("event does not follow correct HEC pre-formatted format for record having offset: {} and topic: {}", record.kafkaOffset(), record.topic());
```

Comment:
We would need to log partition info as well here.

Reply:
Done